lib/controller/controller.go

```diff
@@ -1408,6 +1433,25 @@ func (ctrl *ProvisionController) fetchMountOptions(storageClassName string) ([]s
 	return nil, fmt.Errorf("Cannot convert object to StorageClass: %+v", classObj)
 }
 
+func (ctrl *ProvisionController) fetchAllowedTopologies(storageClassName string) ([]v1.TopologySelectorTerm, error) {
+	classObj, found, err := ctrl.classes.GetByKey(storageClassName)
+	if err != nil {
+		return nil, err
+	}
+	if !found {
+		return nil, fmt.Errorf("StorageClass %q not found", storageClassName)
+	}
+
+	switch class := classObj.(type) {
+	case *storage.StorageClass:
+		return class.AllowedTopologies, nil
+	case *storagebeta.StorageClass:
+		return class.AllowedTopologies, nil
+	}
+
+	return nil, fmt.Errorf("Cannot convert object to StorageClass: %+v", classObj)
```

Comment:
nit: we've been trying to follow the convention of error messages starting with lowercase